packages/editor/src/components/visual-editor/style.scss

```diff
@@ -7,6 +7,13 @@
 	// like the "focused entities".
 	background-color: $gray-300;
 
+	&::before {
+		content: "";
+		position: absolute;
+		inset: 0;
+		background-color: $white;
+	}
```

Comment:
Why is this added a second time? Maybe it needs a comment?

Reply:
Also isn't `.editor-visual-editor` outside the iframe? Why does it need to use a pseudo element here?